kubernetes/apps_kubeclient.go

```diff
@@ -153,21 +167,25 @@ func (kc *kubeClient) GetSpace(spaceName string) (*app.SimpleSpace, error) {
 	// https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/space/create/pipelines/pipelines.component.ts
 	buildconfigs, err := kc.getBuildConfigs(spaceName)
 	if err != nil {
-		return nil, err
+		return nil, errs.WithStack(err)
 	}
 
 	// Get all applications in this space using BuildConfig names
-	apps := make([]*app.SimpleApp, 0)
+	apps := []*app.SimpleApp{}
 	for _, bc := range buildconfigs {
 		appn, err := kc.GetApplication(spaceName, bc)
 		if err != nil {
-			return nil, err
+			return nil, errs.WithStack(err)
 		}
 		apps = append(apps, appn)
 	}
 
 	result := &app.SimpleSpace{
-		Applications: apps,
+		Type: "space",
+		Attributes: &app.SimpleSpaceAttributes{
+			Name:         &spaceName,
+			Applications: apps, // TODO UUID
```

Comment:
what's this `TODO` for exactly ? is it something missing in this PR ?

Reply:
No, it's an old comment. 
OSIO works with UUIDs, k8s works with names; I was hoping to use UUIDs at the time.
I've removed this and similar comments.